README.md

```diff
@@ -8,13 +8,14 @@ CLEO requires an 'ASI Loader' installed to run which is provided with the releas
 No additional files are replaced, however the following files and folders are added:
 
 - cleo\ (CLEO script directory)
-- cleo\.config\sa.json (opcodes info file)
+- cleo\\.config\sa.json (opcodes info file)

```

Comment:
why double slash here?

Reply:
Single slash not displayed in readme

Reply:
Hm, does it mean that all slashes should be escaped?